### DISCUSSION TITLE
S-HOOK EC2 마이그레이션하기

### DISCUSSION THREAD

**Discussion by giscus** [Cyma-s/Cyma-s.github.io#97] [General]
# S-HOOK EC2 마이그레이션하기

이관 과정 DB dump 및 마이그레이션 레포지토리 이관 운영 서버 설정 self-hosted-runner 설정 기존 레포지토리에 러너를 추가한다. Linux, ARM 으로 설정해준다. self-hosted-runner 에 명시되어 있는 대로 진행한다. runner 그룹 이름은 엔터로 하고, 러너 이름을 설정해준다. 라벨은 shook-production …

https://vero.wiki/shook-migration/

**Comment by somsom13** [comment]
너무 멋져요! d=====(￣▽￣*)b

**Reply by Cyma-s** [reply]
@>--------
